Sources/VersionChecker/Networking/ApiFetcher+Extension.swift

```diff
@@ -21,8 +21,13 @@ import Foundation
 import InfomaniakCore
 
 extension Endpoint {
-    static func version(store: Store, platform: Platform, appName: String) -> Endpoint {
-        return Endpoint(path: "/1/app-information/versions/\(store.rawValue)/\(platform.rawValue)/\(appName)")
+    static func version(store: Store, platform: Platform, appName: String, versionType: VersionType) -> Endpoint {
```

Comment:
```suggestion
    static func version(store: Store, platform: Platform, appName: String) -> Endpoint {
```
We don't need VersionType anymore